tests/lsstseries_tests/test_ensemble.py

```diff
@@ -106,14 +107,14 @@ def test_build_index(dask_client):
 @pytest.mark.parametrize("method", ["size", "length", "loglength"])
 @pytest.mark.parametrize("combine", [True, False])
 @pytest.mark.parametrize("sthresh", [50, 100])
-def test_sf2(parquet_ensemble, method, combine, sthresh):
+def test_sf2(parquet_ensemble, method, combine, sthresh, use_map=False):
     """
     Test calling sf2 from the ensemble
     """
 
-    res_sf2 = parquet_ensemble.sf2(combine=combine, method=method, sthresh=sthresh)
+    res_sf2 = parquet_ensemble.sf2(combine=combine, method=method, sthresh=sthresh, use_map=use_map)
     res_batch = parquet_ensemble.batch(
-        calc_sf2, combine=combine, method=method, sthresh=sthresh
+        calc_sf2, use_map=False, combine=combine, method=method, sthresh=sthresh
```

Comment:
Is there a reason why this is explicitly `False` and not using the argument supplied `use_map`

Reply:
Only if you count it being a mistake as a reason 😅 , good catch. It's been fixed now